prisma/seed.ts

```diff
@@ -32,7 +32,7 @@ async function main() {
   //create Label with label name and creator
   const katerinaLabel = await createLabelHelper(
     prisma,
-    'region-east-africa',
+    'region-west-africa',
```

Comment:
🤔

Reply:
I think this is because the migration is creating a `region-east-africa` label so when the seed script is run, we cannot have a duplicate label name?

Reply:
Yep, that is why - I also explained it in the commit message / PR description